src/Database/Driver.php

```diff
@@ -446,4 +446,23 @@ public function __debugInfo()
             'connected' => $this->_connection !== null
         ];
     }
+
+    /**
+     * Truncates an identifier if the database does
+     *
+     * @param string $key Aliased field name
+     *
+     * @return string Aliased field name as returned by Database
+     */
+    public function sanitizeIdentifier(string $key)
```

Comment:
This name is misleading. There isn't any sanitizing being done, could we call this `generateIdentifier()`?

Reply:
yes, done as well